src/checks/trigger-has-unique-ids.js

```diff
@@ -23,7 +23,7 @@ const triggerHasUniqueIds = {
     });
 
     if (doubleId !== undefined) {
-      return [`Got a two or more results with the id of ${doubleId}`];
+      return [`Got a two or more results with the id of (${doubleId})`];
```

Comment:
minor of minor things, but I'd use double quotes stead of parens. 

```
with the id of "5"
```

reads cleaner than 

```
with the id of (5)
```

since we're being literal. Also, lets be clear about why that's an issue. Tack `Expected all trigger ids to be unique` onto the end of the error.

Reply:
Should we change all of the error messages? I used `()` because it was consistent with the others

Reply:
I could go either way. In the other cases, we're putting the whole response object in parens since we're providing general information (rather than saying the object at index 3 is malformed). In this case, we're showing exactly what's wrong (id "5" is not unique), so quotes make more sense to me. 

Either way, super minor. Your call! Merge whatever you like more. If we hate it, ~it's stuck in stone forever~ we'll tweak it further.

Reply:
makes sense - will update to a `"`